robottelo/cli/factory.py

```diff
@@ -3117,7 +3116,8 @@ def configure_env_for_provision(org=None, loc=None):
     puppet_proxy = Proxy.info({'id': Proxy.list()[0]['id']})
     Proxy.update({
         'id': puppet_proxy['id'],
-        'locations': list(set(puppet_proxy['locations']) | {loc['name']}),
+        'locations': list(
+            set(puppet_proxy.get('locations') or []) | {loc['name']}),
```

Comment:
```python
'locations': list(
            set(puppet_proxy.get('locations', []) | {loc['name']}),
```
same for others

Reply:
this solution is giving me indentation error from PEP8. the original indentation is passing flake8 and pep8.

Reply:
this not about indentation but about using 2nd param of get method